src/jsonapi-vuex.js

```diff
@@ -398,6 +398,17 @@ const getters = () => {
       }
       return result
     },
+    getRelated: (state, getters) => (data, seen = []) => {
```

Comment:
Now that both `getRelated` and `followRelationships` go on to call `getRelationships`, I think that the setting of a default value for `seen` can probably be moved to just be in `getRelationships`

Reply:
Looks like it's gone from `getRelated` but also needs removed from `followRelationships` and added to `getRelationships`